unity-client/Assets/ABConverter/Client.cs

```diff
@@ -318,5 +333,151 @@ public static Core.State DumpAsset(string assetHash, string assetFilename, strin
 
             return ConvertAssetToAssetBundle(assetHash, assetFilename, sceneCid, settings);
         }
+        
+        /// <summary>
+        /// Dump all bodyshape wearables normally, including their imported skeleton 
+        /// </summary>
+        public static void DumpAllBodiesWearables()
+        {
+            EnsureEnvironment();
+            
+            List<WearableItem> avatarItemList = GetAvatarMappingList("https://wearable-api.decentraland.org/v2/collections")
+                                                .Where(x => x.category == WearableLiterals.Categories.BODY_SHAPE)
+                                                .ToList();
+
+            Queue<WearableItem> itemQueue = new Queue<WearableItem>(avatarItemList);
+            var settings = new Settings();
+            settings.skipAlreadyBuiltBundles = false;
+            settings.deleteDownloadPathAfterFinished = false;
+            var abConverterCoreController = new ABConverter.Core(ABConverter.Environment.CreateWithDefaultImplementations(), settings);
+            
+            DumpWearableQueue(abConverterCoreController, itemQueue, GLTFImporter_OnBodyWearableLoad);
+        }
+
+        /// <summary>
+        /// Dump all non-bodyshape wearables, optimized to remove the skeleton for the wearables ABs since that is
+        /// only needed for the body shapes (and the WearablesController sets it up for non-bodyshapes in runtime) 
+        /// </summary>
+        public static void DumpAllNonBodiesWearables()
+        {
+            EnsureEnvironment();
+            
+            List<WearableItem> avatarItemList = GetAvatarMappingList("https://wearable-api.decentraland.org/v2/collections")
+                                                .Where(x => x.category != WearableLiterals.Categories.BODY_SHAPE)
+                                                // .Where(x => x.id == "dcl://halloween_2019/vampire_upper_body" 
+                                                            // || x.id == "dcl://base-avatars/bear_slippers"
+                                                            // || x.id == "dcl://exclusive_masks/killer_mask"
+                                                            // ) // For Debugging
```

Comment:
Can we remove this uneeded code? or it was left here for any reason?

Reply:
Done!